lms/templates/word_cloud.html

```diff
@@ -1,30 +1,45 @@
+<%page expression_filter="h"/>
 <%! from django.utils.translation import ugettext as _ %>
 
-<section
+<div
     id="word_cloud_${element_id}"
     class="${element_class}"
     data-ajax-url="${ajax_url}"
 >
+    % if display_name is not display_name_default:
+    <h3 class="hd hd-3">${display_name}</h3>
+    % endif
+
+    % if instructions is not instructions_default:
+    <div class="input_cloud_section" role="group" aria-labelledby="word_cloud_${element_id}_instructions">
+        <div class="input_cloud_instructions" id="word_cloud_${element_id}_instructions">
+            ${instructions}
+        </div>
+    % else:
+    <div class="input_cloud_section" role="group">
+    % endif
 
-    <section class="input_cloud_section">
         % for row in range(num_inputs):
+            <label for="word_cloud_${element_id}_input_${row}" class="sr">${row + 1}/${num_inputs}</label>
```

Comment:
Does this read OK for a screen reader? Would something like "Input 1 out of 5" (translated) be clearer?